test/v1/connectors/LossClaimConnectorSpec.scala

```diff
@@ -157,4 +157,51 @@ class LossClaimConnectorSpec extends ConnectorSpec {
       }
     }
   }
+
+  "delete LossClaim" when {
+
+    "a valid request is supplied" should {
+      "return a successful response with the correct correlationId" in new Test {
+        val expected = Right(DesResponse(correlationId, ()))
+
+        MockedHttpClient
+          .delete(s"$baseUrl/income-tax/claims-for-relief/$nino/$claimId", desRequestHeaders: _*)
+          .returns(Future.successful(expected))
+
+        deleteLossClaimResult(connector) shouldBe expected
+      }
+    }
+
+    "a request returning a single error" should {
+      "return an unsuccessful response with the correct correlationId and a single error" in new Test {
+        val expected = Left(DesResponse(correlationId, SingleError(NinoFormatError)))
+
+        MockedHttpClient
+          .delete(s"$baseUrl/income-tax/claims-for-relief/$nino/$claimId", desRequestHeaders: _*)
+          .returns(Future.successful(expected))
+
+        deleteLossClaimResult(connector) shouldBe expected
+      }
+    }
+
+    "a request returning multiple errors" should {
+      "return an unsuccessful response with the correct correlationId and multiple errors" in new Test {
+        val expected = Left(DesResponse(correlationId, MultipleErrors(Seq(NinoFormatError, LossIdFormatError))))
```

Comment:
LossIdFormat error not actually returned from this endpoint.

Reply:
Fixed thanks